addons/website_slides/controllers/mail.py

```diff
@@ -67,7 +67,8 @@ def mail_update_message(self, res_model, res_id, message, message_id, redirect=N
             domain = [('res_model', '=', res_model), ('res_id', '=', res_id), ('website_published', '=', True), ('message_id', '=', message.id)]
             rating = request.env['rating.rating'].search(domain, order='write_date DESC', limit=1)
             rating.write({
-                'rating': float(post['rating_value'])
+                'rating': float(post['rating_value']),
+                'feedback': html2plaintext(message.body),
```

Comment:
If you look at the patch for this commit, because there wasn't already a trailing comma, you had to change the existing line, which means any changes targeting this line will now conflict when forward porting, all because someone forgot the comma. Using the trailing comma like you did here prevents this problem from happening in the future, so good job and keep doing it :)

Reply:
Thanks :)